src/together/utils/tools.py

```diff
@@ -23,18 +25,62 @@ def normalize_key(key: str) -> str:
     return key.replace("/", "--").replace("_", "-").replace(" ", "-").lower()
 
 
-def parse_timestamp(timestamp: str) -> datetime:
-    formats = ["%Y-%m-%dT%H:%M:%S.%fZ", "%Y-%m-%dT%H:%M:%SZ"]
```

Comment:
Are you sure both of the formats are covered with this logic? I'm concerned about breaking something outside of FT API

Reply:
Reverted